### PR TITLE
build: copy missing code in docker file

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

File: api.Dockerfile [hunk 1]
```diff
@@ -4,12 +4,13 @@ FROM node:lts-alpine AS builder
 WORKDIR /app
 ADD package.json yarn.lock ./
 ADD ./apis/core/package.json ./apis/core/
+ADD ./packages/core/package.json ./packages/core/
```
Comment (tpluscode) [inline api.Dockerfile:7]: Right now workspace packages have to added explicitly

File: api.Dockerfile [hunk 2]
```diff
@@ -4,12 +4,13 @@ FROM node:lts-alpine AS builder
...
 
 # for every new package foo add
 #ADD ./packages/foo/package.json ./packages/foo/
 
 # install and build backend
-RUN yarn install --ci
+RUN yarn install --frozen-lockfile
```
Comment (tpluscode) [inline api.Dockerfile:13]: I figured that `--ci` is not a valid and yarn is not nice enough to point that out 🙄

File: tsconfig.json [hunk 3]
```diff
@@ -7,7 +7,8 @@
     "noImplicitThis": true,
     "noImplicitReturns": true,
     "strictNullChecks": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true
```
Comment (tpluscode) [inline tsconfig.json:11]: Adding temporarily while we don't have proper types for `hydra-box`

**Review by vhf — APPROVED** [review]
👍

**Review by sandhose — APPROVED** [review]
LGTM

**Pull request merged**